cloud_dashboard/src/service/utility.ts

```diff
@@ -287,9 +343,18 @@ export const convertEntityData = (
     };
     for (const launchTemplateColumn of entityColumnList) {
       const rawValue = rawRecord.attributes[launchTemplateColumn.name];
-      dataRecord.value[launchTemplateColumn.name] = launchTemplateColumn.name === 'cloud_context'
-        ? cloudContext.labelName
-        : convertDataForUI(rawValue, launchTemplateColumn, dataCache);
+      if (launchTemplateColumn.name === 'cloud_context') {
+        const temp = cloudContextList.filter((r) => {
+          return r.name !== 'ALL' && r.name === rawValue;
+        });
+        if (temp.length >= 1) {
+          dataRecord.value[launchTemplateColumn.name] = temp[0].labelName;
+        } else {
+          dataRecord.value[launchTemplateColumn.name] = rawValue;
+        }
```

Comment:
@RyoYamashita765 This can be:

```suggestion
        dataRecord.value[launchTemplateColumn.name] = temp.length >= 1
          ? temp[0].labelName
          : rawValue;
```

Reply:
@naoi I've pushed a commit to fix that. Also, the processing in the functions has been organized to improve readability.　Please review it.